Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -67,60 +89,48 @@ public class BellerophonManager: NSObject {
                 // An error occurred
                 completionHandler(result: .Failed)
             }
-        })
+        }
     }
 
     // MARK: Private Methods
-    override init() {
-        super.init()
-        NSNotificationCenter.defaultCenter().addObserver(self, selector: Selector(stopTimer()), name: UIApplicationDidEnterBackgroundNotification, object: nil)
-    }
-
-    deinit {
-        NSNotificationCenter.defaultCenter().removeObserver(self)
-    }
-
-    func handleAppStatus(status: BellerophonStatusProtocol) {
+    private func handleAppStatus(status: BellerophonStatusProtocol) {
         if status.apiInactive() {
-            self.displayKillSwitch(status.userMessage())
```

Comment:
I understand the need to customize user message in the application, but if the application wants to use default user message, which is the user message from server, does the application still need to set/update the label in the killSwitchView, outside Bellerophon? I mean we can add public interfaces that let application to set user message which will replace the ones from the server. When custom user message is not set, it might be better that Bellerophon is going to take care of displaying the message from the server in the killSwitchView.